Makefile

```diff
@@ -0,0 +1,22 @@
+DOCKER := $(shell { command -v podman || command -v docker; })
+
+.PHONY: all clean distclean
+
+all:
+	mkdir -p firmware
+	$(DOCKER) build --tag zmk --build-arg USER_ID=$(shell id -u) .
```

Comment:
Probably should be a more unique tag used for this, TBH.

Reply:
Updated to `zmk-build-user-config`